doc/user/content/install.md

```diff
@@ -54,6 +54,9 @@ curl -L https://downloads.mtrlz.dev/materialized-{{< version >}}-x86_64-apple-da
 
 Run the following commands as root.
 
+**Note!** These instructions changed between versions 0.8.0 and 0.8.1. If you ran them
+previously, you may need to do so again to continue receiving updates.
```

Comment:
hmm this probably should be in a `note` block but that doesn't exist, and it probably doesn't deserve to be in a [`{{ <warning> }}` block](https://github.com/MaterializeInc/materialize/blob/main/doc/user/layouts/shortcodes/warning.html), so I guess this LGTM.

Reply:
I'll wait and see if Sean or Micole has input on how best to format this before landing

Reply:
What is the error folks will run into if they use the "old" command in the new environment?

Reply:
Today they will get v0.8.0 (and never get any new updates); if they run this past July 4th (when our old host is being turned off) they will get a 404 error or similar.

The biggest issue is that people who have already run this in the past will start getting errors too, then navigate to our docs to try to figure out what's going on, and we need to make sure they see this and know they need to run the new instructions.